lib/Math/BivariateCDF.pm

```diff
@@ -0,0 +1,81 @@
+package Math::BivariateCDF;
+
+use 5.014002;
+use strict;
+use warnings;
+use Carp;
+
+require Exporter;
+use AutoLoader;
+
+our @ISA = qw(Exporter);
+
+# Items to export into callers namespace by default. Note: do not export
+# names by default without a very good reason. Use EXPORT_OK instead.
+# Do not simply export all your public functions/methods/constants.
+
+# This allows declaration	use Math::BivariateCDF ':all';
+# If you do not need this, moving things directly into @EXPORT or @EXPORT_OK
+# will save memory.
+our %EXPORT_TAGS = (
+    'all' => [qw(
+            bivnor
+            )]);
+
+our @EXPORT_OK = (@{$EXPORT_TAGS{'all'}});
+
+our @EXPORT = qw(
+
+);
+
+our $VERSION = '0.01_1';
+
+require XSLoader;
+XSLoader::load('Math::BivariateCDF', $VERSION);
+
+# Preloaded methods go here.
+
+# Autoload methods go after =cut, and are processed by the autosplit program.
+
+1;
+__END__
+
+=head1 NAME
+
+Math::BivariateCDF - Perl extension for Bivariate CDF functions.
+
+=head1 SYNOPSIS
+
+  use Math::BivariateCDF;
+
+  Math::BivariateCDF::bivnor( $x, $y, $z );
```

Comment:
It should be like:

```perl
use Math::BivariateCDF qw/bivnor/;

my $variation = bivnor(1, 2, 3);

print("variation = $variation\n"); # should be 0.5

```

I.e. provide a numbers (input arguments) + comment their meaning (or reference to their meaning);